Driverless AI/scoring-pipeline-deployment-to-apache-nifi/scoring-pipeline-deployment-to-apache-nifi.md

```diff
@@ -0,0 +1,352 @@
+
+
+While writing the code for the NiFi processor [ExecuteDaiMojoScoringPipeline.java](https://github.com/james94/dai-deployment-examples/blob/mojo-nifi/mojo-nifi/model-deployment/apps/nifi/nifi-nar-bundles/nifi-h2o-record-bundle/nifi-h2o-record-processors/src/main/java/org/apache/nifi/processors/h2o/record/ExecuteDaiMojoScoringPipeline.java), the following package dependencies for using NiFi's Record Reader, NiFi's Record Writer and H2O's MOJO2 Runtime were required. So the *nifi-h2o-record-processors/* folder's [pom.xml](https://github.com/james94/dai-deployment-examples/blob/mojo-nifi/mojo-nifi/model-deployment/apps/nifi/nifi-nar-bundles/nifi-h2o-record-bundle/nifi-h2o-record-processors/pom.xml) was updated to be able to use these dependencies. Once these new dependencies are added to the *pom.xml*, you must run the following command mvn `eclipse:eclipse -DdownloadSource=true` to download them for use in the Eclipse project. Once all the Java code was written for the processor, the code was compiled with command: `mvn install`. When the code compiled successfully, a Junit test was written in Java [TestExecuteDaiMojoScoringPipeline.java](https://github.com/james94/dai-deployment-examples/blob/mojo-nifi/mojo-nifi/model-deployment/apps/nifi/nifi-nar-bundles/nifi-h2o-record-bundle/nifi-h2o-record-processors/src/test/java/org/apache/nifi/processors/h2o/record/TestExecuteDaiMojoScoringPipeline.java) for testing multiple properties and test cases for the processor. The code was recompiled and a NiFi Archive (NAR) was generated. We will talk more about the NAR soon.
+
+### Add Custom NiFi Processor to NiFi Product
+
+Once you have compiled the NiFi custom processor, a NAR file will be generated. A NAR allows several components and their dependencies to be packaged together into a single package. A NAR package is provided ClassLoader isolation from other NAR packages. So when software from many different organizations is all hosted within the same environment, the Java ClassLoaders will not be an issue.
```

Comment:
confusing sentence: "A NAR package is provided ClassLoader isolation from other NAR packages." 

I am not sure how I will rewrite it